SPECS/grub2/sbat.csv.in

```diff
@@ -1,3 +1,3 @@
 sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
-grub,2,Free Software Foundation,grub,@@VERSION@@,https://www.gnu.org/software/grub/
-grub.azurelinux,2,Microsoft,grub2,@@VERSION_RELEASE@@,https://github.com/microsoft/azurelinux
+grub,4,Free Software Foundation,grub,@@VERSION@@,https://www.gnu.org/software/grub/
+grub.azurelinux,3,Microsoft,grub2,@@VERSION_RELEASE@@,https://github.com/microsoft/azurelinux
```

Comment:
why only 3 for the Microsoft-specific SBAT level?

Reply:
the product-specific sbat level is totally separate from the upstream grub sbat level, so it definitely shouldn't go to `4` just because the upstream grub level is 4.

There is also some "lack of clarity" upstream about how exactly the product-specific sbat should be handled, as far as reset to `1` or bump it (more info is [here](https://github.com/rhboot/shim/issues/634)) but it seems like just bumping the product-specific sbat level is the "safest" thing to do, hence why I bumped it here.